test_runner/fixtures/neon_fixtures.py

```diff
@@ -4962,8 +4962,13 @@ def check_restored_datadir_content(
 
     restored_files = list_files_to_compare(restored_dir_path)
 
+    # pg_notify files are always ignored
+    pgdata_files = [f for f in pgdata_files if not f.startswith("pg_notify")]
+    restored_files = [f for f in pgdata_files if not f.startswith("pg_notify")]
```

Comment:
```suggestion
    restored_files = [f for f in restored_files if not f.startswith("pg_notify")]
```

If I understand it correctly

Reply:
:facepalm: :facepalm: :facepalm: Thank you!

Fixed in 78414c6df